### ISSUE TITLE
Question about experimental results for BERT

### ISSUE THREAD

**Issue by irfan11111111** [qcwthu/Continual_Fewshot_Relation_Learning#5]
Hi, @qcwthu
Thanks for your excellent work about incremental few-shot relation learning. It is really interesting and insightful.
I tried to experiment with BERT, but the accuracy is far from the results in the paper.
Are you welling to open how to run CFRE based on BERT?

**Comment by qcwthu** [comment]
Hi. I  think the key point is to use the pre-trained relational model  and only tune the final layer. Can you try it again?

**Comment by irfan11111111** [comment]
Thanks for your reply.
Does the pre-trained relational model mean your public simmodelckpt? I tried to initialize BERT by simmodelckpt and only tune the final layer. However, the accuracy is still far from the results in the paper.
Can you tell me how to get the pre-trained relational model?

**Comment by qcwthu** [comment]
Hi, Sure! I will upload the sample code or send to you asap. Btw, can you tell me your email address?

**Comment by irfan11111111** [comment]
Thank you! My email address is chenyifan@nudt.edu.cn. Hope to hear from you!

**Comment by dabao12** [comment]
Thanks for your excellent work about incremental few-shot relation learning. It is really interesting and insightful.
I tried to experiment with BERT, but the accuracy is far from the results in the paper.
Are you welling to open how to run CFRE based on BERT?   Whether I can get this part of the code.
My email address is [hanbj890@nenu.edu.cn](mailto:hanbj890@nenu.edu.cn). Hope to hear from you!
